packages/terra-form-fieldset/src/Fieldset.jsx

```diff
@@ -61,12 +62,23 @@ const Fieldset = ({
     { 'legend-visually-hidden': isLegendHidden },
   ]);
 
+  const legendId = `terra-fieldset-legend-${uuidv4()}`;
+  const helpId = `terra-fieldset-help-${uuidv4()}`;
+
+  const addAriadescribedby = (fieldsetChildren) => {
```

Comment:
Can we change function name as it contradicts with it's implementation.  
```suggestion
  const addAriaHelpers = (fieldsetChildren) => {
```